### DISCUSSION TITLE
最新版EDG浏览器无法安装

### DISCUSSION THREAD

**Discussion by y646829569** [the1812/Bilibili-Evolved#1413] [提问 / Q&A] [answered]
EDG浏览器在安装页面点击后，跳出脚本安装界面后闪退，下载内容中出现此文件会危害电脑是否继续的提示，保留双击打开后，显示语法错误，这是为什么？

**Comment by Geeyun-JY3** [comment, marked as answer]
装了脚本管理器没有：https://github.com/the1812/Bilibili-Evolved#脚本管理器

**Reply by y646829569** [reply]
有的是油猴脚本管理器

**Reply by Geeyun-JY3** [reply]
正常来说就不应该下载到本地的。

多试几次？

**Reply by the1812** [reply]
可以下到本地, 然后脚本管理器里新建脚本, 把下好的文件内容粘贴进去

**Reply by y646829569** [reply]
> 正常来说就不应该下载到本地的。
> 
> 多试几次？

![image](https://user-images.githubusercontent.com/26274168/103616127-d09ada80-4f66-11eb-9742-c2b9c1897962.png)
试了很多次，正常脚本安装不是会跳出代码页面以及安装按钮，现在那个代码页面自动弹出，然后EDG下载会完成一个JS文件，双击打开会弹出一个错误页面

**Reply by y646829569** [reply]
> 可以下到本地, 然后脚本管理器里新建脚本, 把下好的文件内容粘贴进去

用记事本打开吗，直接粘贴复制吗？

**Reply by the1812** [reply]
本地双击是无法执行的, 要粘贴到油猴里

**Reply by y646829569** [reply]
![image](https://user-images.githubusercontent.com/26274168/103617067-831f6d00-4f68-11eb-90e3-71f6ddd1c90e.png)
导入显示无法解析

**Reply by y646829569** [reply]
> 本地双击是无法执行的, 要粘贴到油猴里
直接粘贴导致所有代码错行，没有用

**Reply by the1812** [reply]
不是在这里, 是要右上角加号新建一个脚本, 然后把 js 文件的内容记事本打开复制粘贴进去

**Reply by y646829569** [reply]
> 不是在这里, 是要右上角加号新建一个脚本, 然后把 js 文件的内容记事本打开复制粘贴进去

直接粘贴导致所有代码错行，没有用，还是我粘贴的方式不对？

**Reply by the1812** [reply]
![image](https://user-images.githubusercontent.com/26504152/103617899-fd042600-4f69-11eb-8626-6e9d6fc84de7.png)

**Reply by y646829569** [reply]
![image](https://user-images.githubusercontent.com/26274168/103618446-f1652f00-4f6a-11eb-81d6-c8b2bc35197e.png)
粘贴，代码全都在46行了

**Reply by the1812** [reply]
本来就只有46行啊

**Reply by y646829569** [reply]
> 本来就只有46行啊

好了，是我犯傻了。。。谢谢
